.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Qanary component test pipeline
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test-java:
+    runs-on: ubuntu-latest
+    env:
+      BABELFY_API_KEY : someapikey
```

Comment:
@heinpa do I understand it right that this hast to be replaced somehow? It it already done or still in progress?

Reply:
no replacement needed. The components simply need a value for the API key in order to start properly. This can be any random string value, because live tests are disabled and so it's not checked against any endpoint anyway.